apps/frontend/src/app/Types/consts.ts

```diff
@@ -1,14 +1,30 @@
+import { allArtifactSets, allWeaponBowKeys, allWeaponCatalystKeys, allWeaponClaymoreKeys, allWeaponPolearmKeys, allWeaponSwordKeys, nonTravelerCharacterKeys, WeaponBowKey, WeaponCatalystKey, WeaponClaymoreKey, WeaponPoleArmKey, WeaponSwordKey } from "@genshin-optimizer/consts"
+
 export const allHitModes = ["hit", "avgHit", "critHit"] as const
-export const allRegions = ["mondstadt", "liyue", "inazuma", "sumeru", "fontaine", "natlan", "snezhnaya", "khaenriah"] as const
 export const allAmpReactions = ["vaporize", "melt",] as const
 export const allAdditiveReactions = ["spread", "aggravate"] as const
 export const allArtifactSetCount = [1, 2, 3, 4, 5] as const
+/**
```

Comment:
Do we need to have a separate PR that migrates usages of this to the other version, I assume?

Reply:
eventually, the intention here is to only have a single list for `all...` artifact/weapon/character